app/modules/entities/components/editor/property_claims_editor.svelte

```diff
@@ -66,7 +68,21 @@
     {#if isRequiredAndMissing}
       <span class="required-notice">{I18n('required')}</span>
     {/if}
-    <h3 class="editor-section-header">{I18n(customLabel || property)}</h3>
+    <div class="property-info">
+      <h3 class="editor-section-header">{I18n(customLabel || property)}</h3>
+      {#if isWikidataPropertyUri(property)}
+        <a
+          class="uri"
+          href={getWikidataPropertyDocumentationUrl(property)}
```

Comment:
Would it make sense to restrict this display to dataadmin ? to not have the additional ~15-20% vertical space on empty values, for an information that might clutter the UI for users comfortable on the inv editor (and dont care about WD).
Alternatives: 
- make the property title as a link itself (like it happens already to entity label on editor page for example).
- add the link inline, after the property title

Im a bit concerned about liberally adding information to the UI, as we should be careful of keeping it easy to read (this is also a note for future self).

Reply:
actually tweaking some CSS to make it more compact seems enough with 7b0df8292

Reply:
I wonder if we could also just show that info to users who have associated a wikidata account to there user account

Reply:
neat, this is a usecase that could fit as a new role `wikidatauser` which would be updated after oauth autorisation/revocation

Reply:
I don't think we need a new `role` for that, that's just inferred from the state of `user.oauth.wikidata`. That could also be a user display setting that is either automatically turned on when connecting a wikidata account, or offered to be turned on at that moment(?)

Reply:
A "Display setting" sounds good, as some users might want access to Wikidata documentation without OAuth-ing they're user account.